engines/dradis-api/config/routes.rb

```diff
@@ -1,11 +1,17 @@
 Dradis::CE::API::Engine::routes.draw do
```

Comment:
I added some changes to this file in this PR that points to this same branch: https://github.com/dradis/dradis-ce/pull/385/files#diff-0f2e3d02b44405293aedb7d1ee196292